sc2reader/factories/plugins/replay.py

```diff
@@ -100,21 +100,30 @@ def APMTracker(replay):
     above actions divided by the number of seconds played by the player (not
     necessarily the whole game) multiplied by 60.
     """
+
+    speed_multiplier = 1
+    if replay.expansion == 'LotV':
+        speed_multiplier = 1.4
+
+    game_seconds_per_second = 1.4
+    if replay.expansion == 'LotV':
+        game_seconds_per_second = 1.4
```

Comment:
I just took the output from the test and set it as expected, since I don't have a good understanding of what the avg_apm actually has to be because of rounding and series manipulation (the 0th minute is excluded for some reason).

I wasn't able to get it equal to in-game match screen without tweaking numbers and I didn't want to just tweak a number to multiply with without understanding why, so I tried to get it as close as possible while still understanding how. 😄

Should we try to get the same numbers as in-game (at least for average)?